### ISSUE TITLE
Temperature line for ground storage items is missing from translations files and crowdin

### ISSUE THREAD

**Issue by tannyl** [anegostudios/VintageStory-Issues#4607]
### Game Version

1.20.0-rc.1

### Platform

Linux

### Modded

Vanilla

### SP/MP

Singleplayer

### Description

The text string used for displaying temperature on ground storage items (when they are over 20C) is missing from the translations files and crowdin. The string is `Temperature: {0:0.##}°C`

The string in the code is encapsulated in `Lang.Get()` so it seems that the string is simply just missing from the files.

The string is found here: https://github.com/anegostudios/vssurvivalmod/blob/f905b60d6967cf9758689df723886230205b7260/BlockEntity/BEGroundStorage.cs#L1156

### Screenshots

![Screenshot from 2024-12-05 22-06-08](https://github.com/user-attachments/assets/025fd6dc-f494-4721-9dae-1a30eeb414c8)

### Logs

```Log```

**Comment by Craluminum2413** [comment]
It is translatable but no one translated it

**Comment by radfast** [comment]
Thank you.  Added to 1.20-rc.6  (or before that if we update CrowdIn in advance).  I've used the key `temperature-precise`, noting the point that it's easier for translators if new keys are language neutral.

So in `en.json` we will now have:

	"Temperature: {0}°C": "Temperature: {0}°C",
	"temperature-precise": "Temperature: {0:0.#}°C",
	"+{0:0.#}°C": "+{0:0.#}°C",

In terms of the number of decimal places, I didn't like the previous inconsistency, I can't see any reason why we would ever need more than 0.1 °C of precision in the game.  Bearing in mind as well the game is on an alternative Mediaeval timeline, and even modern thermometers, unless they are high precision digital devices found in science laboratories, do not have more than 0.1 °C of precision.
